### PR TITLE
Refactor step by step 

### PR DESCRIPTION
**Pull request by HeleneGrini**
- Written in typescript
- `Step`, `Content` and `Descriptions` were perviously properties of the `StepByStep` class. These are now exported as components from index.ts and accessed from `@telia/styleguide` as `Step`, `StepContent` and `StepDescription`
- Enable customizing of icon, header and content in `StepDescription`
- Add different sizes for a StepByStep component
- Support dashed borders
- Disable animations for non-interactive StepByStep components

### CODE REVIEW

File: src/molecules/StepByStep/StepIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  /**
+   * index of the step
+   */
+  index?: number;
+}
+
+export const StepIcon: React.FC<Props> = (props) => {
+  const { iconClassName, onSelect, index, interactive, iconName, imageSrc, number, opened, pebbles } = props;
+
+  const elementType = interactive ? 'a' : 'div';
```
Comment (mariannemagnussen) [inline src/molecules/StepByStep/StepIcon.tsx:30]: Blir det riktig å bruke en a tag når det ikke er en link, men et onClick event ?

**Review by mariannemagnussen — APPROVED** [review]
Ser meget bra ut 👌  Et spørsmål bare

File: src/molecules/StepByStep/StepIcon.tsx (hunk 1 above)
Comment (HeleneGrini) [reply src/molecules/StepByStep/StepIcon.tsx:30]: nei 😂  Good catch

**Review by mariannemagnussen — APPROVED** [review]
nice 👌

**Pull request merged**